toolchains/syno-armada38x-7.0/Makefile

```diff
@@ -0,0 +1,11 @@
+TC_ARCH = armada38x
+TC_VERS = 7.0%20preview
+TC_KERNEL = 3.10.108+
```

Comment:
For the DSM7 I assume the kernel version is higher than `4.4.xxx+`

Reply:
```
$ uname -a
Linux NAS 4.4.180+ #41222 SMP Fri Dec 4 19:02:12 CST 2020 x86_64 GNU/Linux synology_apollolake_418play
```

Reply:
NVM They did not update the kernel for this arch :shrug:  https://sourceforge.net/projects/dsgpl/files/Tool%20Chain/DSM%207.0%20preview%20Tool%20Chains/Marvell%20Armada%2038x%20Linux%203.10.108/